cli/QuickstepCli.cpp

```diff
@@ -440,6 +462,13 @@ int main(int argc, char* argv[]) {
     }
   }
 
+#ifdef QUICKSTEP_ENABLE_GOOGLE_PROFILER
+  if (!query_num && !quickstep::FLAGS_profile_file_name.empty()) {
```

Comment:
You're right. I'll fix. The profiler produces an output even if you don't explicitly stop the profiler, which is why I didn't catch it. Good catch!